Cargo.toml

```diff
@@ -16,6 +16,13 @@ include = [
     "README.md"
     ]
 
```

Comment:
Can you add yourself to authors and bump the version to 0.0.5 as well?

Reply:
Thank you for confirming. I updated these.